app/src/main/kotlin/com/bitwarden/authenticator/data/authenticator/manager/model/VerificationCodeItem.kt

```diff
@@ -20,7 +20,7 @@ data class VerificationCodeItem(
     val issueTime: Long,
     val id: String,
     val issuer: String?,
-    val accountName: String?,
+    val label: String?,
```

Comment:
I like the change. It's definitely an improvement. Can we make sure iOS is using similar terminology?

Just FYI, this was originally `username`. In #19 we changed it to `accountName`. IIRC, it was done to align with already established terminology. Since this is not user facing though, I think it's worth cleaning up to align with industry terms instead of internal ones.

Reply:
So the iOS app is using `name` and `username`: https://github.com/bitwarden/authenticator-ios/blob/e50d6242dd3a1fbe7cc51e1f43a140ab4d26398e/AuthenticatorShared/Core/Vault/Models/Domain/AuthenticatorItem.swift#L52-L58

Does that mean I should revert the name changes in this PR to match iOS? Just my two cents, but I think using `label` and `issuer` is significantly less confusing.

Reply:
We talked offline and agreed to keep what we have here because it is clearer, even if it differs form what is in the iOS codebase.